File: 프로그래머스/0/120817. 배열의 평균값/배열의 평균값.cpp
#include <string>
#include <vector>

using namespace std;

double solution(vector<int> numbers) {
    double answer = 0;
    
    double sum = 0;
    for (int num : numbers)
        sum += num;
    
    answer = sum / numbers.size();
    
    return answer;
}

File: 프로그래머스/0/120817. 배열의 평균값/배열의 평균값.swift
import Foundation

func solution(_ numbers:[Int]) -> Double {
    //let sum = numbers.reduce(0, +)
    
    var sum = 0;
    for i in 0..<numbers.count {
        sum += numbers[i]
    }
    
    return Double(sum) / Double(numbers.count)
}